src/__tests__/compression.js

```diff
@@ -46,68 +42,3 @@ describe('compressData()', () => {
         expect(given.subject).toMatchSnapshot()
     })
 })
-
-describe('Payload Compression', () => {
```

Comment:
removing as it's already covered by the tests in `decide.js` + `posthog-core.js`